packages/mobile/src/geth/geth.ts

```diff
@@ -149,38 +153,36 @@ async function initGeth() {
   }
 }
 
-export async function getGeth(): Promise<typeof gethInstance> {
+export async function getGeth(sync: boolean = true): Promise<typeof gethInstance> {
   Logger.debug('Geth@getGeth', 'Getting Geth Instance')
   if (!gethInstance) {
-    await initGeth()
+    await initGeth(sync)
   }
   return gethInstance
 }
 
-async function ensureStaticNodesInitialized(): Promise<boolean> {
+async function ensureStaticNodesInitialized(sync: boolean = true): Promise<boolean> {
   const { nodeDir } = networkConfig
-  if (await staticNodesAlreadyInitialized(nodeDir)) {
-    Logger.debug('Geth@maybeInitStaticNodes', 'static nodes already initialized')
-    return true
-  } else {
-    Logger.debug('Geth@maybeInitStaticNodes', 'initializing static nodes')
-    let enodes: string | null = null
-    try {
-      enodes = await StaticNodeUtils.getStaticNodesAsync(DEFAULT_TESTNET)
-    } catch (error) {
-      Logger.error(
-        `Failed to get static nodes for network ${DEFAULT_TESTNET},` +
-          `the node will not be able to sync with the network till restart`,
-        error
-      )
-      return false
-    }
-    if (enodes != null) {
-      await writeStaticNodes(nodeDir, enodes)
-      return true
-    }
+  Logger.debug('Geth@ensureStaticNodesInitialized', 'initializing static nodes')
+  let enodes: string | null = null
+  try {
+    enodes = await StaticNodeUtils.getStaticNodesAsync(DEFAULT_TESTNET)
```

Comment:
we don't need to fetch these if sync is disabled, right? we can just set it to `[]`